app.json

```diff
@@ -32,7 +32,7 @@
     },
     "orientation": "portrait",
     "icon": "./assets/images/icon.png",
-    "userInterfaceStyle": "light",
+    "userInterfaceStyle": "automatic",
```

Comment:
If you set `userInterfaceStyle` to `automatic` here, you should remove `"userInterfaceStyle": "automatic"` from lines 46 and 50.
```json
"userInterfaceStyle": "automatic",
```

Reply:
On this Android device, the minimum mode does not work, this is better